README.md

```diff
@@ -1,20 +1,20 @@
 # Search Engine Parser
 
-<span><i>"If it is a search engine, then it can be parsed"</i> - Some random guy</span>
+<span><i>"If it is a search engine, then it can be parsed"</i> - some random guy</span>
 
 ![Demo](assets/animate.gif)
 
 [![Python 3.5|3.6|3.7|3.8](https://img.shields.io/badge/python-3.5%7C3.6%7C3.7%7C3.8-blue)](https://www.python.org/downloads/)
-[![PyPI version](https://badge.fury.io/py/search-engine-parser.png)](https://badge.fury.io/py/search-engine-parser)
```

Comment:
Why change the source of the badge?

Reply:
Old one was pretty blurry, also just fits better

Reply:
Alright